packages/design-system/src/styled-props/background.css

```diff
@@ -506,6 +506,34 @@
     background: var(--primary-bg);
   }
 
+  &.background-corporateCdBackground {
```

Comment:
You can probably drop the `Background` suffix on these

Reply:
That's fair, I just wanted to match the naming convention from all the other colors above